### PR TITLE
[Docs] Update model serving docs

### PR DESCRIPTION
**Pull request by jond01**
I reviewed these files and used this opportunity to fix some docs issues related to model serving and monitoring.

### CODE REVIEW

File: docs/serving/custom-model-serving-class.md [hunk 1]
```diff
@@ -173,10 +173,21 @@ is used to create real-time dashboards, detect drift, and analyze performance.
...
+```
 
+Optional arguments:
 * **stream_path** &mdash; Enterprise: the v3io stream path (e.g. `v3io:///users/..`); CE: a valid Kafka stream 
 (e.g. kafka://kafka.default.svc.cluster.local:9092)
 * **sample** &mdash; optional, sample every N requests
 * **batch** &mdash; optional, send micro-batches every N requests
-* **tracking_policy** &mdash; optional, model tracking configurations, such as setting the scheduling policy of the model monitoring batch job
+
+Before deploying a model with model monitoring enabled via `fn.set_tracking()`,
```
Comment (jillnogold) [inline docs/serving/custom-model-serving-class.md]: Before you deploy a model with model monitoring enabled via `fn.set_tracking()`,
set the credentials for the project:

File: docs/serving/custom-model-serving-class.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ See the full [Model Server example](https://github.com/mlrun/functions/blob/mast
 If you want to use multiple versions for the same model, use `:` to separate the name from the version. 
 For example, if the name is `mymodel:v2` it means model name `mymodel` version `v2`.
 
-You should specify the `model_path` (url of the model artifact/dir) and the `class_name` name 
+You should specify the `model_path` (URL of the model artifact/dir) and the `class_name` name
```
Comment (jillnogold) [inline docs/serving/custom-model-serving-class.md]: You should specify the `model_path` (URL of the model artifact/dir) and the `class_name`

Comment (jond01) [reply docs/serving/custom-model-serving-class.md]: Done in ae3ac18

File: docs/serving/custom-model-serving-class.md (hunk 1 above)
Comment (jond01) [reply docs/serving/custom-model-serving-class.md]: Done in 3d0a97f

**Review by jillnogold — APPROVED** [review]

**Review by assaf758 — APPROVED** [review]

**Pull request merged**